airflow-core/src/airflow/ui/src/layouts/Details/DetailsLayout.tsx

```diff
@@ -92,50 +112,76 @@ export const DetailsLayout = ({ children, error, isLoading, tabs }: Props) => {
               {dagView === "graph" ? <Graph /> : <Grid limit={limit} />}
             </Box>
           </Panel>
-          <PanelResizeHandle
-            className="resize-handle"
-            onDragging={(isDragging) => {
-              if (!isDragging) {
-                const zoom = getZoom();
+          {!isRightPanelCollapsed && (
+            <PanelResizeHandle
+              className="resize-handle"
+              onDragging={(isDragging) => {
+                if (!isDragging) {
+                  const zoom = getZoom();
 
-                void fitView({ maxZoom: zoom, minZoom: zoom });
-              }
-            }}
-          >
-            <Box bg="fg.subtle" cursor="col-resize" h="100%" transition="background 0.2s" w={0.5} />
-          </PanelResizeHandle>
-          <Panel defaultSize={dagView === "graph" ? 30 : 80} minSize={20}>
-            <Box display="flex" flexDirection="column" h="100%">
-              {children}
-              {Boolean(error) || (warningData?.dag_warnings.length ?? 0) > 0 ? (
-                <>
-                  <ActionButton
-                    actionName={translate("common:dagWarnings")}
-                    colorPalette={Boolean(error) ? "red" : "orange"}
-                    icon={<LuFileWarning />}
-                    margin="2"
-                    marginBottom="-1"
-                    onClick={onOpen}
-                    rounded="full"
-                    text={String(warningData?.total_entries ?? 0 + Number(error))}
-                    variant="solid"
-                  />
+                  void fitView({ maxZoom: zoom, minZoom: zoom });
+                }
+              }}
+            >
+              <Box
```

Comment:
Do we need the `<Box>` here, or is just the `<Button>` enough?

Reply:
![image](https://github.com/user-attachments/assets/043932fb-1886-4050-8066-fc6f3ca7357e)
Hi @guan404ming, thanks for your kind suggestion.
I think `<Box>` starting at line 126 is necessary since it aligns the `<IconButton>` to center.
I tested UI without it. As the image shows, the IconButton would be in another position, and the resize line (`<PanelResizeHandle>`) would disappear although its functionality remains. (You can still drag the line to resize the right panel although the line disappears.)
Thus, I think we still need the `<Box>`.

Reply:
Thanks for the test, let's keep it and keep the width consistent to 0.5
```suggestion
                w={0.5}
```

Reply:
Sorry for the mistake and thank you for pointing it out. I've edited it in commit "[change Box width](https://github.com/apache/airflow/pull/51946/commits/04641454b882a48076b32bcfd7bf78514dbcb606)"